EndlessClient/Rendering/Character/CharacterAnimationActions.cs

```diff
@@ -415,6 +415,14 @@ private bool IsSteppingStone(CharacterRenderProperties renderProps)
                 || _currentMapProvider.CurrentMap.Tiles[renderProps.GetDestinationY(), renderProps.GetDestinationX()] == TileSpec.Jump;
         }
 
+        public void NotifyAdminHideEffect(int playerId)
```

Comment:
Move this method up to where the rest of the public interface is defined, general style is to keep public/private methods grouped together in a file

Reply:
@ethanmoffat done. Sorry it took a month :P